### ISSUE TITLE
SimpleSAMLphp

### ISSUE THREAD

**Issue by jestinijames** [geerlingguy/drupal-vm#1484]
### Summary

Hi Jeff. I am working with drupalVM on windows 10.  This is not an issue per se. I just wanted some help. So you can close it as soon as you see it. I have been trying to setup SimpleSamlphp on my system for a while now. But it seems extremely complex. I tried following instructions from two different sites:
http://valuebound.com/resources/blog/how-to-configure-single-sign-on-across-multiple-drupal-8-platforms-or-websites
https://www.chapterthree.com/blog/how-to-configure-simplesamlphp-for-drupal-8-on-acquia
But I've kept getting stuck.

I was wondering if you could maybe make a short video on how to setup SimpleSAMLphp on windows 10 using drupalVM. Your previous videos have all been extremely helpful. Or atleast point me in the right direction atleast in terms of instructions.

if you want to know where i usually get stuck is, when i try to add '/simplesaml/' to the url, I get a forbidden page. 
Thanks  in advance

**Comment by geerlingguy** [comment]
@jestinijames - Yeah... SimpleSAMLphp is usually pretty complicated :(

I've only implemented it one time in the past, and it took about a week for me to get everything up and running. I don't think I'd be able to be much of a help as I don't have the experience to make it easy and digestible.

However, I'd point you to BLT's SimpleSAMLphp example documentation, which is what I used to get things configured the first (and only) time I did it: http://blt.readthedocs.io/en/8.x/readme/simplesamlphp-setup/

**Comment by sbubaron** [comment]
I know this is a few years late, but having just moved to a new computer and needing to go through the whole setup again, I found myself also getting the 403 forbidden error with simplesaml. 

My original solution (https://github.com/geerlingguy/drupal-vm/issues/708) using mklink /D to create a symlink wasn't working and I wasted an afternoon trying to figure it out. 

This morning I decided to start from scratch and ran vagrant destroy than vagrant up and it worked! So I guess the symlink needs to be created before running vagrant up (vagrant reload --provision) didn't do it for me. 

Anyway, take this as proof that simplesaml can work in drupal-vm 5.1 on Windows 10 (host)
